shuup/front/template_helpers/general.py

```diff
@@ -24,6 +24,13 @@
 from shuup.front.utils.views import cache_product_things
 from shuup.utils.mptt import get_cached_trees
 from shuup.utils.translation import cache_translations_for_tree
+from shuup.utils.importing import cached_load
+
+
+def get_login_fields(request):
```

Comment:
change the of the function. it doesn't return login fields.

Reply:
Fixed